README.rst

```diff
@@ -0,0 +1,114 @@
+courseraresearchexports
+=======================
+
+.. image:: https://travis-ci.org/coursera/courseraresearchexports.svg
+    :target: https://travis-ci.org/coursera/courseraresearchexports
+
+This project is a library consisting of a command line interface and a client
+for interacting with Coursera's research exports.
+
+Installation
+------------
+
+To install this sdk, simply execute::
+
+    sudo pip install courseraresearchexports
+
+`pip <https://pip.pypa.io/en/latest/index.html>`_ is a python package manager.
+If you do not have ``pip`` installed on your machine, please follow the
+installation instructions for your platform found at:
+https://pip.pypa.io/en/latest/installing.html#install-or-upgrade-pip
+
+Setup
+-----
+
+Authorize your application using `courseraoauth2client <https://github.com/coursera/courseraoauth2client>`_::
+
+    courseraoauth2client config authorize --app manage-research-exports
+
+Command Line Interface
+----------------------
+
+The project includes a command line tool. Simply run::
+
+    courseraresearchexports -h
+
+for a complete list of features, flags, and documentation.
+
+jobs
+^^^^
+
+create
+~~~~~~
+
+Create an data export job request and return the export job id.  There are two
+general workflows for creating data export requests.  To create a data export
+requests for all available schemas for a course::
+
+    courseraresearchexports jobs create schemas --courseId $COURSE_ID --statementOfPurpose "testing data export"
+
+If a more limited set of data is required, you can specify which schemas are
+included with the export.  (e.g. for the demographics tables)::
+
+    courseraresearchexports jobs create schemas --courseId $COURSE_ID --schemas demographics --statementOfPurpose "testing data export"
+
+If you are a data coordinator, you can request that user ids are linked between
+domains of the data export::
+
+    courseraresearchexports jobs create schemas --courseId $COURSE_ID --statementOfPurpose "testing data export" --anonymityLevel HASHED_IDS_NO_PII
```

Comment:
can you refer to the gitbook section that outlines all the schemas available? it's available here: https://coursera.gitbooks.io/data-exports/content/introduction/programmatic_access.html